bigbluebutton-html5/imports/ui/components/poll/service.js

```diff
@@ -95,7 +95,9 @@ const getPollResultsText = (isDefaultPoll, answers, numRespondents, intl) => {
     const pctBars = "|".repeat(pct * MAX_POLL_RESULT_BARS / 100);
     const pctFotmatted = `${Number.isNaN(pct) ? 0 : pct}%`;
     if (isDefaultPoll) {
-      const translatedKey = intl.formatMessage(pollAnswerIds[item.key.toLowerCase()]);
+      const translatedKey = pollAnswerIds[item.key.toLowerCase()] 
+        ? intl.formatMessage(pollAnswerIds[item.key.toLowerCase()])
```

Comment:
Maybe we should include some extra validation here:
- item.key exists
- intl.formatMessage has the id defined by pollAnswerIds[item.key.toLowerCase()]

Reply:
Item.key is the poll answers so they always exists
All pollAnswerIds have defined ids in the en.json file
Do you think it is needed anyway?

Reply:
Hmmm, I guess not if we guarantee those values. I'll set this as resolved. Thanks @germanocaumo